src/dlgPackageExporter.cpp

```diff
@@ -131,12 +131,27 @@ dlgPackageExporter::~dlgPackageExporter()
     delete ui;
 }
 
-void dlgPackageExporter::appendToConfigFile(QString& comment, const QString& what, const QString& value)
+void dlgPackageExporter::appendToCommentAndConfigFile(const QString& what, const QString& value)
```

Comment:
I don't feel like this change is worth it - doesn't improve too much but does make it lengthier.

Reply:
The method now acts on its members (`(QString) mPackageConfig` and `(QString) mPackageComment` - the latter is new in this PR) rather than accepting a `QString` reference to whatever. And since it is now working on two elements instead of just one it made sense (to me at least) to rename it to reflect that...

Reply:
I didn't see that logic... let's go with the shorter name

Reply:
`appendToWhat` though? We are adding to both the config file text and the comment text...

If you insist on a shorter name then I will choose `appendToDetails(...)` ...

Reply:
Yeah or `appendToComment`?